tests/testNRyield.cc

```diff
@@ -108,8 +132,16 @@ int main(int argc, char* argv[]) {
   G4int points  = (argc>4) ? atoi(argv[4]) : 100;
   G4String mat  = (argc>5) ? argv[5] : "G4_Si";
 
+  // Default values for Emp Lindhard model
+  G4double klow = (argc>6) ? strtod(argv[6],0) : 0.040;
+  G4double khigh = (argc>7) ? strtod(argv[7],0) : 0.142;
+  G4double Elow = (argc>8) ? strtod(argv[8],0)*keV : 0.39*keV;
+  G4double Ehigh = (argc>9) ? strtod(argv[9],0)*keV : 7.0*keV;
+  G4bool EDepK = (argc>10) ? (std::stoi(argv[10])!=0) : true;
+  G4double kFixed = (argc>11) ? strtod(argv[11],0) : 0.158;
+ 
```

Comment:
See my comment in G4CMPEmiricalNIEL.  If you put these numerical defaults directly into G4CMPConfigManager then you can replace the numbers here with calls to G4CMPConfigManager.

That avoids the problem of having hardcoded numbers in multiple places and future inconsistencies when only one of them gets changed.